.github/workflows/ci.yml

```diff
@@ -19,8 +19,9 @@ jobs:
     - name: Install ckb-debugger
       run: |
         cd ..
-        git clone https://github.com/nervosnetwork/ckb-standalone-debugger
+        git clone https://github.com/mohanson/ckb-standalone-debugger
```

Comment:
Suggest installing 2 versions of ckb-debugger. The official one is to do tests and the `spawn` one is for demo.